### PR TITLE
[Breaking] add runtime options in wgpu init methods

### PR DESCRIPTION
**Pull request by nathanielsimard**
Add the ability to control the Runtime options in both wasm and normal Rust.

### CODE REVIEW

File: crates/burn-wgpu/src/runtime.rs [hunk 1]
```diff
@@ -51,16 +51,52 @@ impl<G: GraphicsApi, F: FloatElement, I: IntElement> Runtime for WgpuRuntime<G,
...
+    pub dealloc_strategy: DeallocStrategy,
+    /// Control the slicing strategy.
+    pub slice_strategy: SliceStrategy,
+    /// Control the amount of compute tasks to be aggregated into a single GPU command.
+    pub max_tasks: usize,
+}
+
+impl Default for RuntimeOptions {
+    fn default() -> Self {
+        let max_tasks = match std::env::var("BURN_WGPU_MAX_TASKS") {
```
Comment (antimora) [inline crates/burn-wgpu/src/runtime.rs:66]: Can we build conditionally initialize this environment variable depending it's wasm or not?

Comment (nathanielsimard) [reply crates/burn-wgpu/src/runtime.rs:66]: no_std isn't supported for wgpu, but this will work with wasm.

Comment (antimora) [reply crates/burn-wgpu/src/runtime.rs:66]: Got it. Feels quick and dirty though =D

I'll approve it for now.

**Review by antimora — APPROVED** [review]

File: crates/burn-wgpu/src/runtime.rs (hunk 1 above)
Comment (nathanielsimard) [reply crates/burn-wgpu/src/runtime.rs:66]: This option might even go away in future releases

**Pull request merged**